backend/src/data/seeds/fill-db.ts

```diff
@@ -0,0 +1,12 @@
+import { Knex } from 'knex';
+import { operationSystem } from '~/data/seed-data/operation-system.seed';
+import { TableName } from '~/common/enums/db/table-name.enum';
+
+async function seed(knex: Knex): Promise<void> {
```

Comment:
actually this is a part of our application (it isn't a mock data). since it is, it should be a migration instead of a seed